packages/openapi/src/openapi-request-builder.spec.ts

```diff
@@ -125,7 +125,7 @@ describe('openapi-request-builder', () => {
 
     const timeoutAboveDelay = () =>
       new OpenApiRequestBuilder('get', '/with-delay')
-        .middleware([timeout(delayInResponse * 10)])
+        .middleware(timeout(delayInResponse * 10))
```

Comment:
[pp] Instead of testing this implicitly, you could have a separate test to see whether both variants work. I am not sure if we now have a test that checks `[]` as well. Then again we already have tests for `transformVariadicArgumentToArray()`. So maybe this is not necessary.

Reply:
Added two tests for both overloads.